### PR TITLE
TASK: Lazy load images

### PR DESCRIPTION
**Pull request by Sebobo**
This change adds the layzr lazy load library which
loads images when necessary.

Img tags can have the following data-attributes to allow lazyloading:

* data-image-normal (required)
* data-image-retina
* data-image-srcset

The normal src attribute should be set to use a simple loading gif.

Resolves: #122

### CODE REVIEW

**Comment by Sebobo** [comment]
@bwaidelich this solution works really well for images, I'll test if it works with gifs.
The showcase went down to 1,3mb instead of 27,5mb on initial load.

**Comment by Sebobo** [comment]
Have to test it a bit more in the backend, maybe have to disable it there.

**Comment by bwaidelich** [comment]
Sweet!

**Comment by Sebobo** [comment]
Ok this works well now. Disabled it in the backend.
Gif are also lazy loaded.

File: Packages/Sites/Neos.NeosIo/Resources/Public/Styles/Main.css [hunk 1]
```diff
@@ -2940,7 +2940,7 @@ sub {
  * References:
  * Layout for show cases
  */
-.references__list .references__item:nth-child(odd) {
```
Comment (kdambekalns) [inline Packages/Sites/Neos.NeosIo/Resources/Public/Styles/Main.css]: What? How can that be correct?

File: Packages/Sites/Neos.NeosIo/Resources/Public/Styles/Main.css [hunk 2]
```diff
@@ -2940,7 +2940,7 @@ sub {
...
+.references__list .references__item:nth-ch(odd) {
```
Comment (kdambekalns) [inline Packages/Sites/Neos.NeosIo/Resources/Public/Styles/Main.css]: I mean, how can _this_ be correct…

Comment (Sebobo) [inline Packages/Sites/Neos.NeosIo/Resources/Public/Styles/Main.css]: Thx for spotting that! I accidentally broke that in another PR.

**Pull request merged**